src/main/java/org/opendevstack/provision/services/BitbucketAdapter.java

```diff
@@ -534,12 +534,12 @@ public static boolean containsGroup(JsonNode json, String group) {
         repo.setAdminGroup(project.getProjectAdminGroup());
```

Comment:
This seems now to not be covered with a unit test. Can you please extend it?

Reply:
Done